### PR TITLE
adapted readme and examples

### PR DESCRIPTION
**Pull request by MarionBla**
Adaptation of the readme and examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +34,49 @@ AND (alias n°2).d = "value 2"
...
+    "alias n°2":{
+        "c":"value 1",
+        "d":"value 2"
+    },
+    "alias n°3":{
+        "a":"value 3",
+        "b":"value 4"
+    }
+}}
+```
```
Comment (SaulLu) [inline README.md:79]: As discussed, in order to gain impact, I think we could:
* rework the sentence "The SQL query is written in a parser and transformed into a URL to retrieve data from a FHIR API. In the end, the JSON type data will be returned as tabular data."
* create a "Getting Started" section that says there are 2 methods to use the module: string queries and config file queries. These 2 examples could be explained in 2 sub-sections in which there would be a "functional" code (which works by copy and paste).

File: examples/example2.md [hunk 2]
```diff
@@ -1,68 +1,72 @@
...
-> **Question:** Get the pressure measures of patients born after 1970, together with their gender and birthdate
+> **Question:** Get clinical information about patients that had a general examination because of Coronavirus
+>
+> Here we get diagnosis codes for finished encounters. We get the period of the encounter and some information about the patient such as his age, gender and if he is alive.
 
-Note that http://loinc.org|55284-4 corresponds to blood pressure.
+Note that we use the following SNOMED CT codes:
+- 162673000: General examination of patient (procedure)
+- 840546002: Exposure to severe acute respiratory syndrome coronavirus 2 (event)
+- _(840539006: Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder))_
```
Comment (LaRiffle) [inline examples/example2.md:10]: we're not using this code 840539006, can we rm it?

File: README.md [hunk 3]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
...
+
+Two possible ways to enter the query : as a SQL query or as a JSON config file
+
+**SQL query as entry**
+
+```
+import pandas as pd
+from fhir2dataset.query import Query
+from fhir2dataset.fhirrules_getter import FHIRRules
+from fhir2dataset.parser import FHIR2DatasetParser
```
Comment (SaulLu) [inline README.md:33]:
```suggestion
from fhir2dataset import Query, FHIRRules, FHIR2DatasetParser
```

File: README.md [hunk 4]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
...
+
+
+## Getting started
+
+Two possible ways to enter the query : as a SQL query or as a JSON config file
+
+**SQL query as entry**
+
+```
+import pandas as pd
```
Comment (SaulLu) [inline README.md:30]: I wonder if we can do without the `import pandas`, what do you think?

File: examples/example2.md (hunk 2 above)
Comment (MarionBla) [reply examples/example2.md:10]: Yes indeed, thank you !

File: README.md [hunk 5]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
...
+```
+```
+# Enter in dirname the path of config.json
+filename_config = 'config.json'
+
+with open(os.path.join(dirname, filename_config)) as json_file:
+    config = json.load(json_file)
+
+query.from_config(config)
+query.execute()
```
Comment (SaulLu) [inline README.md:119]: Maybe we could put forward the recovery of the table resulting from the query, what do you think?

```suggestion
query.execute()
df = query.main_dataframe
```

File: README.md [hunk 6]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
...
 AND (alias n°3).a = "value 3"
-AND (alias n°3).b = "value 4"
+AND (alias n°3).b = "value 4""
 ```
 
-## Installation
+```
+config_from_parser = parser.parse(sql_like_query)
+query.from_config(config_from_parser)
+query.execute()
```
Comment (SaulLu) [inline README.md:55]: Maybe we could put forward the recovery of the table resulting from the query, what do you think? (idem below)

```suggestion
query.execute()
df = query.main_dataframe
```

File: README.md [hunk 7]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
 
 This project is still under development.
 
-This repo is a POC allowing to make a query (close to SQL format) on a FHIR API and to retrieve tabular data.
+This repo is a POC allowing to make a SQL query on a FHIR API and to retrieve tabular data. 
 
-The request that FHIR2Dataset performs on a FHIR API is specified in a configuration file of this [form](examples/config_template.json).
-The purpose of this file is to perform a query whose result will be identical to the next SQL query:
+## Installation
+
```
Comment (SaulLu) [inline README.md:12]: As we aim to publish the new version on PyPI, we could be announced that it's possible to install it with `pip`!

```suggestion
### With pip

```
pip install fhir2dataset
```

### From source
```

File: README.md [hunk 8]
```diff
@@ -6,41 +6,126 @@ Transform FHIR to dataset for ML applications
 
 This project is still under development.
 
-This repo is a POC allowing to make a query (close to SQL format) on a FHIR API and to retrieve tabular data.
+This repo is a POC allowing to make a SQL query on a FHIR API and to retrieve tabular data.
```
[the remark this answers is not in the document]
Comment (MarionBla) [reply README.md]: Yes thank you! I changed it, what do you think of it now ?

File: README.md (hunk 4 above)
Comment (MarionBla) [reply README.md:32]: Yes it is true thanks ! In the demo tutorial we can delete it too, don't you think ?

File: README.md (hunk 5 above)
Comment (MarionBla) [reply README.md:119]: Yes it is much better !

File: README.md (hunk 4 above)
Comment (SaulLu) [reply README.md]: Yes!  Great point, we can delete it, it's useless :smile:

File: README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (LaRiffle) [reply README.md]: Great!

**Pull request merged**